ultralytics/hub/session.py

```diff
@@ -93,7 +93,7 @@ def load_model(self, model_id):
         """Loads an existing model from Ultralytics HUB using the provided model identifier."""
         self.model = self.client.model(model_id)
         if not self.model.data:  # then model does not exist
-            raise ValueError(emojis("❌ The specified HUB model does not exist"))  # TODO: improve error handling
```

Comment:
Should we still raise an error here? 

Otherwise an error will be raised later that may seem unrelated to a missing model no?